.internal-ci/helm/fog-services/templates/fog-view-store-svc.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "fogServices.fullname" . }}-fog-view-store
+  labels:
+    app: fog-view
+    {{- include "fogServices.labels" . | nindent 4 }}
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    app: fog-view-store
+    {{- include "fogServices.selectorLabels" . | nindent 4 }}
+  ports:
+    - name: view
```

Comment:
Follow patterns from fog-view-service.yaml: 

```
- name: view-grpc
  port: 3225
  targetPort: view-grpc
- name: mgmt-http
    port: 8000
    targetPort: mgmt-http
  - name: view-http
    port: 8225
    targetPort: view-http
```

Reply:
Will handle in follow-up PR.